### PR TITLE
Add mozorg/about/governance/policies/reporting.ftl

### PR DESCRIPTION
**Pull request by amychurchwell**
Ref: https://github.com/mozilla/bedrock/pull/9708

### CODE REVIEW

**Comment by amychurchwell** [comment]
@peiying2 may I have your review for en before adding all the other locales?

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+reporting-this-document-is-intended = This document is intended as an interface to existing documents, processes and people responsible for ensuring { -brand-name-mozilla }’s communities are healthy, and inclusive for all.
+reporting-when-to-report = When To Report
+reporting-please-report-all-incidents = Please report all incidents where someone has engaged in behavior that is potentially illegal or makes you or someone else feel unsafe, unwelcome or uncomfortable <a href="{ $cpg }">as further explained in the CPG</a>.
+reporting-how-to-give-a-report = How to Give a Report
+reporting-if-you-believe-someone = If you believe someone is in physical danger call your local emergency number.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:16]: Insert a comma after `danger` so it reads `... physical danger, call your ...`

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+reporting-if-you-have-a-report-by = If you have a report <strong>by <em>and</em> about</strong> a contributor (for example, the report is made <strong>by</strong> one contributor <strong>about</strong> another contributor), then you should make your report at the <a href="{ $community_hotline }">Community Participation Guidelines hotline</a>.
+reporting-if-you-have-a-report-involving = If you have a report <strong>involving an employee, contractor, or vendor</strong> (for example, the report is made <strong>by</strong> an employee or is <strong>about</strong> an employee) then you should report at the <a href="{ $employee_hotline }">{ -brand-name-mozilla } Employee hotline</a>.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:18]: Add a comma after `an employee` so it reads `... <strong>about</strong> an employee), then you should...`

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+reporting-contractor = Contractor
+reporting-vendor = Vendor
+reporting-if-someone-reports-to = If someone reports to you…
+reporting-do-not-question-or-judge = Do not question, or judge their experience.
+reporting-do-not-invite-them-to = Do not invite them to withdraw the incident report.
+reporting-do-not-promise-any-particular = Do not promise any particular response.
+reporting-do-let-them-know-that = <strong>Do</strong> let them know that for { -brand-name-mozilla }’s policy to be impactful, reports should go through the hotline. If they do not feel comfortable filing the report themselves, you may do so.
+reporting-no-matter-who-files-the = No matter who files the report, the information below is important to capture.
+reporting-names-of-the-people-involved = Names of the people involved (or if names are unknown, use descriptions and any identifiable info such as appearance, role, handle, project/community affiliation).
+reporting-description-of-incident = Description of incident, including memorable dates (or event) and locations.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:35]: Should `(or event)` be plural as dates and locations are plural?

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+reporting-if-the-reporter-wants = If the reporter wants to make an anonymous report, please inform them that this contact information we may not be able to update the initial reporter if appropriate. Some laws prohibit anonymous reporting and that you may be required to provide their name if you are a { -brand-name-mozilla } Manager or Community Leader.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:36]: This part needs to be rewritten, it doesn't make any sense: `... report, please inform them that this contact information we may not be able to update the initial reporter if appropriate.`

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+reporting-relationship-of-reportervictim = Relationship of reporter/victim.
+reporting-mozilla-managers-and-community = { -brand-name-mozilla } Managers and Community leaders
+reporting-if-a-mozilla-manager-or = If a { -brand-name-mozilla } Manager or Community leaders is informed about potential <a href="{ $cpg }">CPG</a> violations they are expected to immediately report the incident through the applicable hotline, even if the initial reporter will also file a report. { -brand-name-mozilla } Managers and Community leaders are not permitted to investigate complaints on their own.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:39]: - `leaders` should be singular `or Community leader is ...`
- Insert a comma before `they` so it reads `CPG</a> violations, they ...`

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+reporting-do-not-impose-your-own = Do not impose your own judgement on how the reporter should react. { -brand-name-focus } on listening.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:40]: - `Focus` here is a verb, not a placeholder
- This string appears to be bold in the staging. I don't see any markup to indicate so.

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+reporting-what-happens-after-the = What happens after the Report is filed
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:41]: Lower case R as in `report`

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+reporting-investigation = Investigation
+reporting-reports-are-handled-discretely = Reports are handled discretely and privately, and will only be shared with the people who can investigate, respond, and advise. As part of this investigation, it may be necessary for some information to be disclosed to others, for example to key stakeholders administering communities or events, witnesses, and the wrongdoer.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:43]: - Insert a comma after `for example, to key ... `
- Plural for `wrongdoers` as the other nouns are plural.

**Review by peiying2 — CHANGES_REQUESTED** [review]
@amychurchwell Other than the specific comments, here are two general feedback
- Header styles are inconsistent:
  - This is a title
  - This Is a Title
- Missing URL links thorough out the doc.

File: en/mozorg/about/governance/policies/reporting.ftl (hunk 4 above)
Comment (amychurchwell) [reply en/mozorg/about/governance/policies/reporting.ftl:36]: Going to try and find someone to help us with the wording for this one. This is (unfortunately) blocked until we do.

**Comment by Francescaminelli** [comment]
Thank you for pointing out that one sentence was not clear, I will try to clarify it as soon as possible. 

Unfortunately I did not re-check the whole document, but only underlined what changes needed to be made due to the changes in reporting system. My understanding was that the rest of the document would have not be changed at all from the original page. 

(could you confirm that these pages were not changed at all, apart from the changes I asked for?
https://www.mozilla.org/en-US/about/governance/policies/participation/
https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/community-hotline/
https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/)

**Comment by Francescaminelli** [comment]
Hi, thank you for this.

The sentence can be changed to : "please inform them that without a contact information we may not be able to update the initial reporter if appropriate". 

Concerning the missing URL link, I am not sure what links are missing. They should be left the same as the original page: https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/ .

**Comment by peiying2** [comment]
@amychurchwell the set of CPG related pages are offered to limited locales too, so we need to add the config file to this as well. Let's resolve the technical issue first before adding more pages to Pontoon.

**Comment by peiying2** [comment]
> Concerning the missing URL link, I am not sure what links are missing. They should be left the same as the original page: https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/ .

Sometimes I see files call out { url } and the link itself proceeding the localizable string. I don't see it here. If this is how it's structured, I am ok. I don't know how they are all connected, that's all.

**Comment by amychurchwell** [comment]
@peiying2 the urls have different variable names in this file and the link itself is passed to the fluent helper bedrock-side.

**reporting.ftl:** `reporting-please-report-all-incidents = Please report all incidents where someone has engaged in behavior that is potentially illegal or makes you or someone else feel unsafe, unwelcome or uncomfortable <a href="{ $cpg }">as further explained in the CPG</a>.`

**bedrock:** `{{ ftl('reporting-please-report-all-incidents', cpg=url('mozorg.about.governance.policies.participation')) }}`

File: en/mozorg/about/governance/policies/reporting.ftl [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+reporting-do-not-promise-any-particular = Do not promise any particular response.
+reporting-do-let-them-know-that = <strong>Do</strong> let them know that for { -brand-name-mozilla }’s policy to be impactful, reports should go through the hotline. If they do not feel comfortable filing the report themselves, you may do so.
+reporting-no-matter-who-files-the = No matter who files the report, the information below is important to capture.
+reporting-names-of-the-people-involved = Names of the people involved (or if names are unknown, use descriptions and any identifiable info such as appearance, role, handle, project/community affiliation).
+reporting-description-of-incident = Description of incident, including memorable dates (or events) and locations.
+reporting-if-the-reporter-wants = If the reporter wants to make an anonymous report, please inform them that without a contact information we may not be able to update the initial reporter if appropriate. Some laws prohibit anonymous reporting and that you may be required to provide their name if you are a { -brand-name-mozilla } Manager or Community Leader.
```
Comment (peiying2) [inline en/mozorg/about/governance/policies/reporting.ftl:36]: This sentence needs a polish! 
- `...without a contact information` is incorrect. Either `a contact` or `contact information` (drop the `a`). 
- Insert a comma after `contact information`.
- There are two `ifs` in the full sentence. Maybe changing the 2nd `if` to `when appropriate`?

**Review by peiying2 — CHANGES_REQUESTED** [review]
The corrected sentence still has some issues. Otherwise, the rest and the config file look good.

File: en/mozorg/about/governance/policies/reporting.ftl (hunk 9 above)
Comment (amychurchwell) [reply en/mozorg/about/governance/policies/reporting.ftl]: done!

**Comment by peiying2** [comment]
I meant to approve it first. Thanks for the fix.

**Pull request merged**